src/test/java/org/xwiki/git/script/GitScriptServiceTest.java

```diff
@@ -54,6 +54,52 @@
     public ComponentManagerRule componentManager = new ComponentManagerRule();
 
     private File testRepository;
+    
+    private void getGitRepositoryAndFindAuthors(String username, String accessCode) throws Exception
+    {
+        GitScriptService service = this.componentManager.getInstance(ScriptService.class, "git");
+        Repository repository;
+        if (username == null && accessCode == null) {
+            repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED);
+        } else {
+            repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED,
+                username, accessCode);
+        }
```

Comment:
You haven't changed this yet. You don't need the "if" here anymore (see my previous message). Thx

Reply:
I tried that and ran into coverage errors during maven build.

Reply:
See: ![vmplayer_KClfMISxpB](https://user-images.githubusercontent.com/33906649/79054959-a845b000-7c62-11ea-949c-f752cf62ece4.png)

Reply:
From what I remember from googling, I believe this is related META-INF..
just found it here again: [link](https://github.com/unitsofmeasurement/unit-api/issues/161)
Let me try some solutions with this.

Reply:
Actually, no, the error is different now (built again).
Now it's 0.86 instead of 0.0 (and fails because less than 0.87):
![image](https://user-images.githubusercontent.com/33906649/79055125-e42d4500-7c63-11ea-95bd-17308fa4daa2.png)

Reply:
Ok now I know, doing the changes and trying again.

Reply:
Hm still same problem. (ratio is 0.86)
What I did: I just added `username, accessCode` in the parameters.
Like:
```
repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED, "test user", "TestAccessCode");
```
Whereas the whole java file is unchanged.

Reply:
Changes commited so you can see the code.